projects/chaos/inc/charger.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+// Charger module for interacting with the charger board over CAN.
+//
+// Requires CAN to be enabled.
+
+#include "exported_enums.h"
+#include "status.h"
+
+// Configure the charger and set up CAN handlers.
+StatusCode charger_init(void);
+
+// Sets the charger state.
+// - If enabled it will only be active when the charger is connected to the charger board.
```

Comment:
By enabled/disabled do you mean relay closed/open? I'm a little confused by the terms.

Reply:
Done.